--- scite-1.61/scite/gtk/makefile.orig	2004-07-29 21:14:01.293547008 +0000
+++ scite-1.61/scite/gtk/makefile	2004-07-29 21:13:27.303714248 +0000
@@ -45,7 +45,7 @@
 bindir=$(prefix)/bin
 SYSCONF_PATH=$(prefix)/share/scite
 
-INSTALL=install -o root -g root
+INSTALL=install
 
 PROG	= ../bin/SciTE
 
@@ -59,9 +59,9 @@
 CXXBASEFLAGS=-W -Wall -DGTK -DSCI_LEXER -DPIXMAP_PATH=\"$(pixmapdir)\" -DSYSCONF_PATH=\"$(SYSCONF_PATH)\" $(INCLUDEDIRS)
 
 ifdef DEBUG
-CXXTFLAGS=-DDEBUG -g $(CXXBASEFLAGS)
+CXXTFLAGS=-DDEBUG $(OPTFLAGS) $(CXXBASEFLAGS)
 else
-CXXTFLAGS=-DNDEBUG -Os $(CXXBASEFLAGS)
+CXXTFLAGS=-DNDEBUG $(OPTFLAGS) $(CXXBASEFLAGS)
 endif
 
 ifndef NO_LUA
@@ -97,13 +97,10 @@
 deps:
 	$(CC) -MM `$(CONFIGFLAGS)` $(CXXFLAGS) *.cxx ../src/*.cxx >deps.mak
 
-# make should be run in ../../scintilla/gtk to compile all the lexers.
-COMPLIB=../../scintilla/bin/scintilla.a
-
 $(PROG): SciTEGTK.o SciTEBase.o SciTEBuffers.o SciTEIO.o Exporters.o \
 MultiplexExtension.o DirectorExtension.o SciTEProps.o Utf8_16.o \
-	$(COMPLIB) $(LUA_OBJS)
-	$(CC) `$(CONFIGLIB)` `$(CONFIGTHREADS)` -DGTK $^ -o $@
+	$(LUA_OBJS)
+	$(CC) `$(CONFIGLIB)` `$(CONFIGTHREADS)` -DGTK $^ -o $@ -lscintilla
 
 # Automatically generate header dependencies with "make deps"
 include deps.mak
@@ -125,6 +122,7 @@
 ifndef GTK2
 	$(INSTALL) -m 644 -D SciTE.desktop $(DESTDIR)$(datadir)/gnome/apps/Applications/SciTE.desktop
 endif
+	$(INSTALL) -m 644 -D SciTE.desktop $(DESTDIR)$(datadir)/applications/SciTE.desktop
 	$(INSTALL) -m 644 -D Sci48M.png $(DESTDIR)$(pixmapdir)/Sci48M.png
 endif
 
