--- scite-1.76/scite/gtk/SciTE.desktop.orig	2006-08-28 03:49:27.000000000 +0200
+++ scite-1.76/scite/gtk/SciTE.desktop	2008-03-30 15:22:36.000000000 +0200
@@ -1,16 +1,16 @@
 [Desktop Entry]
 Encoding=UTF-8
-Version=1.0
 Name=SciTE Text Editor
 Name[es]=Editor de textos SciTE
 Name[fr]=Éditeur de texte SciTE
 Comment=Edit your source files
 Comment[es]=Edita especializado para programación
 Comment[fr]=Éditer des fichiers sources
-GenericName=Text Editor
-Type=Application
+Comment[pl]=Edytor tekstu SciTE
 Exec=SciTE %F
 Icon=Sci48M.png
 Terminal=false
+Type=Application
 StartupNotify=true
-Categories=TextEditor;Development;GTK;Application;
+Categories=GTK;Development;
+# vi: encoding=utf-8
