diff -urN scite162.org/scite/gtk/SciTE.desktop scite162/scite/gtk/SciTE.desktop
--- scite162.org/scite/gtk/SciTE.desktop	2004-08-20 09:21:00.000000000 +0200
+++ scite162/scite/gtk/SciTE.desktop	2004-11-23 22:21:39.874847929 +0100
@@ -1,12 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
-Version=1.0
 Name=SciTE
-GenericName=Text Editor
-Type=Application
 Comment=SciTE Text Editor
+Comment[pl]=Edytor tekstu SciTE
 Exec=SciTE
 Icon=Sci48M.png
 Terminal=false
+Type=Application
 MultipleArgs=false
-Categories=GNOME;Application;
+Categories=GTK;TextEditor;
+# vi: encoding=utf-8
