--- scite-3.5.1/scite/gtk/SciTE.desktop.orig	2014-11-15 20:18:17.155715011 +0100
+++ scite-3.5.1/scite/gtk/SciTE.desktop	2014-11-15 20:19:35.615711719 +0100
@@ -1,19 +1,19 @@
 [Desktop Entry]
 Encoding=UTF-8
-Version=1.0
 Name=SciTE Text Editor
 Name[es]=Editor de textos SciTE
 Name[fr]=Éditeur de texte SciTE
+Name[pl]=Edytor tekstu SciTE
 Name[ru]=Текстовый редактор SciTE
 Comment=Edit your source files
 Comment[es]=Edita especializado para programación
 Comment[fr]=Éditer des fichiers sources
+Comment[pl]=Edycja plików źródłowych
 Comment[ru]=Редактор исходных кодов
 GenericName=Text Editor
-Type=Application
 Exec=SciTE %F
 Icon=Sci48M
 Terminal=false
 StartupNotify=true
-Categories=TextEditor;Development;GTK;Application;
+Categories=GTK;Development;
 MimeType=text/plain;
