diff -urN scite162.org/scite/gtk/makefile scite162/scite/gtk/makefile
--- scite162.org/scite/gtk/makefile	2004-08-20 09:21:00.000000000 +0200
+++ scite162/scite/gtk/makefile	2004-11-23 22:20:05.481069583 +0100
@@ -45,7 +45,7 @@
 bindir=$(prefix)/bin
 SYSCONF_PATH=$(prefix)/share/scite
 
-INSTALL=install -o root -g root
+INSTALL=install
 
 PROG	= ../bin/SciTE
 
@@ -59,9 +59,9 @@
 CXXBASEFLAGS=-W -Wall -DGTK -DSCI_LEXER -DPIXMAP_PATH=\"$(pixmapdir)\" -DSYSCONF_PATH=\"$(SYSCONF_PATH)\" $(INCLUDEDIRS)
 
 ifdef DEBUG
-CXXTFLAGS=-DDEBUG -g $(CXXBASEFLAGS)
+CXXTFLAGS=-DDEBUG $(OPTFLAGS) $(CXXBASEFLAGS)
 else
-CXXTFLAGS=-DNDEBUG -Os $(CXXBASEFLAGS)
+CXXTFLAGS=-DNDEBUG $(OPTFLAGS) $(CXXBASEFLAGS)
 endif
 
 ifndef NO_LUA
@@ -102,8 +102,8 @@
 
 $(PROG): SciTEGTK.o SciTEBase.o SciTEBuffers.o SciTEIO.o Exporters.o \
 MultiplexExtension.o DirectorExtension.o SciTEProps.o Utf8_16.o \
-	$(COMPLIB) $(LUA_OBJS)
-	$(CC) `$(CONFIGTHREADS)` -DGTK $^ -o $@ `$(CONFIGLIB)`
+	$(LUA_OBJS)
+	$(CC) `$(CONFIGTHREADS)` -DGTK $^ -o $@ `$(CONFIGLIB)` -lscintilla
 
 # Automatically generate header dependencies with "make deps"
 include deps.mak
@@ -127,6 +127,7 @@
 else
 	$(INSTALL) -m 755 -D SciTE.desktop $(DESTDIR)$(datadir)/gnome/apps/Applications/SciTE.desktop
 endif
+	$(INSTALL) -m 644 -D SciTE.desktop $(DESTDIR)$(datadir)/applications/SciTE.desktop
 	$(INSTALL) -m 644 -D Sci48M.png $(DESTDIR)$(pixmapdir)/Sci48M.png
 endif
 
